components/automate-ui/src/app/entities/users/user.effects.ts

```diff
@@ -65,11 +65,10 @@ export class UserEffects {
       });
     }));
 
-  // TODO rename
   @Effect()
   getUser$ = combineLatest([
     this.actions$.pipe(ofType<GetUser>(UserActionTypes.GET)),
-    this.store$.select(iamMajorVersion)])
+    this.store$.select(iamMajorVersion).pipe(filter(identity))])
```

Comment:
without this, a `nil` was sneaking into the getUser request initially, resulting in a call to the v1 API by default, followed by a v2 call once `iamMajorVersion` was actually populated